service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -40,22 +42,72 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             run3093(caseDetails);
         }
 
+        if ("FPLA-3175".equals(migrationId)) {
+            run3175(caseDetails);
+        }
+
         caseDetails.getData().remove(MIGRATION_ID_KEY);
         return respond(caseDetails);
     }
 
+    private void run3175(CaseDetails caseDetails) {
+        final String familyManCaseNumber = "CV21C50026";
+
+        Map<UUID, String> documentsToRemove = Map.of(
+            UUID.fromString("339ad24a-e83c-4b81-9743-46c7771b3975"),
+            "Children's Guardian Position Statement 22 .06.2021",
+
+            UUID.fromString("dc8ad08b-e7ba-4075-a9e4-0ff8fc85616b"),
+            "Supporting documents for Children's Guardian Position Statement no 1",
+
+            UUID.fromString("008f3053-d949-4aaa-9d65-1027e88ed288"),
+            "Supporting documents for Children's Guardian Position Statement no 2",
+
+            UUID.fromString("671d6805-c042-4b3e-88e8-7c7fd103a026"),
+            "Supporting documents for Children's Guardian Position Statement no 3",
+
+            UUID.fromString("74c918f2-6409-4651-bb7b-1f58c39aee94"),
+            "Supporting documents for Children's Guardian Position Statement no 4",
+
+            UUID.fromString("03343373-df27-4744-a63c-56a98442662a"),
+            "Supporting documents for Children's Guardian Position Statement no 5"
+        );
+
+        CaseData caseData = getCaseData(caseDetails);
+        validateFamilyManNumber("FPLA-3175", familyManCaseNumber, caseData);
+
+        List<Element<CourtAdminDocument>> otherCourtAdminDocuments = caseData.getOtherCourtAdminDocuments();
+
+        for (Map.Entry<UUID, String> entry : documentsToRemove.entrySet()) {
```

Comment:
Can just be `documentsToRemove.entrySet().forEach(entry -> removeOtherCourtAdminDocument(otherCourtAdminDocuments, entry));`